src/components/ServerDetails/ServerDetails.Props.ts

```diff
@@ -0,0 +1,51 @@
+export interface IServerDetailsProps {
+    serverStatus: string;
+    networkData: INetworkData;
+    diskData: IDiskData;
+    memoryData: IMemoryData;
+    headerData: IHeaderData;
+    cpuData: ICpuData;
+}
+
+export interface IHeaderData {
+    serverName: string;
+    fqdmServerName: string;
```

Comment:
This is Martin's component that I merged with my branch for further development. But He managed to change it, so I'm guessing it will be up soon